tests/behaviour/contracts/copy_memory_to_calldata/dynArray.sol

```diff
@@ -35,71 +35,68 @@ function returnUint256() pure external returns (uint256[] memory) {
     return y;
 }
 
+function returnStruct() pure external returns (structDef2[] memory) {
+    structDef2[] memory y = new structDef2[](3);
+    y[0] = structDef2(1,10);
+    y[1] = structDef2(2,20);
+    y[2] = structDef2(3,30);
+
+    return y;
+}
+
+function returnNestedStruct() pure external returns (structDef1[] memory) {
+    structDef1[] memory y = new structDef1[](3);
+    y[0] = structDef1(1,structDef2(10, 100), 1000);
+    y[1] = structDef1(2,structDef2(20, 200), 2000);
+    y[2] = structDef1(3,structDef2(30, 300), 3000);
+
+    return y;
+}
+
+function returnMultipleFeltDynArray() pure external returns (uint8[] memory, uint8[] memory) {
+     uint8[] memory x = new uint8[](4);
+     x[0] = 5;
+     x[1] = 6;
+     x[2] = 7;
+     x[3] = 8;
+
+     uint8[] memory y = new uint8[](3);
+     y[0] = 1;
+     y[1] = 2;
+     y[2] = 3;
+
+     return (y, x);
+}
+
+function returnDynArrayAsMemberAccess() pure external returns (uint[] memory){
+    uint256[] memory y = new uint256[](2);
+    y[0] = 10;
+    y[1] = 100;
+   // y[2] = 1000;
```

Comment:
Could you delete this line in the next PR that is rebased off this one? I don't want the CI to have to run again.